requirements.txt

```diff
@@ -0,0 +1,15 @@
+jax 
+jaxlib
+flax
+torch
+transformers
+diffusers
+numpy
+optax
+torchvision
+datasets
+Pillow
+wandb
+tqdm
+huggingface_hub
+accelerate
```

Comment:
I noticed that a few packages from `requirements_flax.txt` are missing here, but I don't know whether we'd use them or not.

```
chex
orbax
ftfy
tensorboard
Jinja2
```

Reply:
Let's leave them out for now and add them if we get errors. I don't no a better way, but having unnecessary libs can lead to more conflicts down the road.